bindata/network/multus/cni-plugins-unsupported.yaml

```diff
@@ -0,0 +1,44 @@
+---
+kind: DaemonSet
+apiVersion: apps/v1
+metadata:
+  name: containernetworking-cni-plugins-unsupported
+  namespace: multus
+  annotations:
+    kubernetes.io/description: |
+      This daemon set launches *unsupported* CNI plugins on each node.
```

Comment:
if they're unsupported then shouldn't the user have to deal with them themsevles?

Reply:
@danwinship yeah, I don't know; Marc and the NFV team decided to do this kind of split.